my-project/src/store/index.js

```diff
@@ -1,8 +1,49 @@
 import { createStore } from 'vuex';
+import db from '../plugins/firebase.config';
 
 export default createStore({
-  state: {},
-  mutations: {},
-  actions: {},
+  state: {
+    loginUserData: {},
```

Comment:
どのようなプロパティが格納されるのかがわかるように、予め初期値を入れておくと良いでしょう。
以下のイメージです。
```suggestion
    loginUserData: {
      userName: ''
    },
```